src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo;
+  closeModel: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, closeModel }) => {
```

Comment:
The naming convention for the `closeModel` method does not follow the best practices for event handler functions in React. Consider renaming it to `onCloseModal` or `handleCloseModal` ([source](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)).